pkgs/applications/editors/neovim/gnvim/default.nix

```diff
@@ -31,13 +31,13 @@ rustPlatform.buildRustPackage rec {
         f.write_all(b"const VERSION: &str = \"${version}\";").unwrap();
     }
     EOF
-  '';
 
-  buildPhase = ''
-    make build
+    # Install the binary ourselves, since the Makefile doesn't have the path
+    # containing the target architecture
+    sed -e "/target\/release/d" -i Makefile
```

Comment:
This shouldn't be in prePatch but postPatch instead. Otherwise patches don't apply cleanly.

Reply:
Thanks!